ReactiveSwift.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,1595 @@
+				4A0E10FF1D2A92720065D310 /* Lifetime.swift in Sources */,
+				D0C312E719EF2A5800984962 /* Scheduler.swift in Sources */,
+				D0C312CD19EF2A5800984962 /* Atomic.swift in Sources */,
+				D08C54BA1A69C54300AD8286 /* Property.swift in Sources */,
+				D0D11AB91A6AE87700C1F8B1 /* Action.swift in Sources */,
+				C79B647C1CD52E23003F2376 /* EventLogger.swift in Sources */,
+				9ABCB1851D2A5B5A00BCA243 /* Deprecations+Removals.swift in Sources */,
+				D08C54B81A69A9D000AD8286 /* SignalProducer.swift in Sources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+		D04725F119E49ED7006002AA /* Sources */ = {
+			isa = PBXSourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				D0A2260E1A72F16D00D33B74 /* PropertySpec.swift in Sources */,
+				B696FB811A7640C00075236D /* TestError.swift in Sources */,
+				D021671D1A6CD50500987861 /* ActionSpec.swift in Sources */,
+				D0C3130E19EF2B1F00984962 /* SchedulerSpec.swift in Sources */,
+				BFA6B94D1A7604D400C846D1 /* SignalProducerNimbleMatchers.swift in Sources */,
+				D8170FC11B100EBC004192AD /* FoundationExtensionsSpec.swift in Sources */,
+				C79B64741CD38B2B003F2376 /* TestLogger.swift in Sources */,
+				CA6F28501C52626B001879D2 /* FlattenSpec.swift in Sources */,
+				4A0E11041D2A95200065D310 /* LifetimeSpec.swift in Sources */,
+				CDCD247A1C277EEC00710AEE /* AtomicSpec.swift in Sources */,
+				579504331BB8A34200A5E482 /* BagSpec.swift in Sources */,
+				D0A226081A72E0E900D33B74 /* SignalSpec.swift in Sources */,
+				02D2602B1C1D6DB8003ACC61 /* SignalLifetimeSpec.swift in Sources */,
+				D0C3130C19EF2B1F00984962 /* DisposableSpec.swift in Sources */,
+				D0A2260B1A72E6C500D33B74 /* SignalProducerSpec.swift in Sources */,
+				D8024DB21B2E1BB0005E6B9A /* SignalProducerLiftingSpec.swift in Sources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+		D047260719E49F82006002AA /* Sources */ = {
+			isa = PBXSourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				D08C54B41A69A2AF00AD8286 /* Signal.swift in Sources */,
+				D8E84A671B3B32FB00C3E831 /* Optional.swift in Sources */,
+				D0C312D419EF2A5800984962 /* Disposable.swift in Sources */,
+				D08C54B91A69A9D100AD8286 /* SignalProducer.swift in Sources */,
+				9ABCB1861D2A5B5A00BCA243 /* Deprecations+Removals.swift in Sources */,
+				EBCC7DBD1BBF01E100A2AE92 /* Observer.swift in Sources */,
+				D85C652B1C0E70E3005A77AD /* Flatten.swift in Sources */,
+				4A0E11001D2A92720065D310 /* Lifetime.swift in Sources */,
+				D08C54BB1A69C54400AD8286 /* Property.swift in Sources */,
+				D03B4A3E19F4C39A009E02AC /* FoundationExtensions.swift in Sources */,
+				D000040A1A46864E000E7D41 /* TupleExtensions.swift in Sources */,
+				D08C54B71A69A3DB00AD8286 /* Event.swift in Sources */,
+				C79B647D1CD52E4A003F2376 /* EventLogger.swift in Sources */,
+				D0C312CE19EF2A5800984962 /* Atomic.swift in Sources */,
+				D0C312E819EF2A5800984962 /* 
```

Comment:
This is the list of files in the target. I don't think they can be inherited.

(Unless GitHub messed up your comment, which seems possible since they don't show up on the diff page.)